src/fs.rs

```diff
@@ -194,73 +155,66 @@ mod tests {
         assert!(!is_included(&path));
     }
 
+    fn fn_exclude(ex: Vec<globset::Glob>) -> globset::GlobSet {
+        let mut build = globset::GlobSetBuilder::new();
+        for x in ex {
+            build.add(x);
+        }
+        build.build().expect("bad")
```

Comment:
Nit: can we add more expansive expectations here? Like `.expect("Failed to build GlobSet")`?

Reply:
With the changes in f4660bd, this .expect is no longer needed. (or really it's now a `?` to waterfall errors up)